src/mocha-env-loader.js

```diff
@@ -1,38 +1,40 @@
-var sourceMap = require('source-map')
-var loaderUtils = require('loader-utils')
+'use strict';
 
-var SourceNode = sourceMap.SourceNode
-var SourceMapConsumer = sourceMap.SourceMapConsumer
+const sourceMap = require('source-map');
+const loaderUtils = require('loader-utils');
 
-module.exports = function(content, map) {
-  this.cacheable()
+const SourceNode = sourceMap.SourceNode;
+const SourceMapConsumer = sourceMap.SourceMapConsumer;
 
-  var sourceNode
-  var id = this.options.name
+module.exports = function (content, map) {
+  this.cacheable();
```

Comment:
This is no longer necessary and can be removed as loaders are cached by default since `webpack >= v2.0.0`